aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -68,6 +72,10 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.ristretto255.scalar_sub, "ristretto255.scalar_sub", 1],
     [.ristretto255.scalar_parse_arg, "ristretto255.scalar_parse_arg", 1],
 
+    [.bulletproofs.base, "bulletproofs.base", 1],
```

Comment:
Shouldn't these be declared optional to avoid breaking the VM (and thus the entire network) in case the native functions make it to a node software update before we can update the onchain gas schedule? cc @vgao1996

Reply:
Right, forgot about that. Thank you for catching this! @vgao1996 and @wrwg hopefully there are mechanisms in place to prevent fools like me from taking down the network.

Reply:
> hopefully there are mechanisms in place to prevent fools like me from taking down the network?

We are incrementally building those protections. We now have a test for framework compatibility, but I don't think this covers this problem with gas,

In general I'm not sure whether 5 days before major code freeze we should land something like this.

@davidiw

Reply:
We won't; no worries!